docs/.vitepress/utils/term-parser.ts

```diff
@@ -0,0 +1,39 @@
+export default [
+  {
+    regex: /:([\w+]*):([\w+]*):/g,
+    modifier: (match: RegExpMatchArray) => {
+      console.log(match)
+      const m1 = match[1].replace(/_/g, " ");
+      const m2 = match[2].replace(/_/g, " ");
+      return `<Term term="${m1}" show="${m2}"/>`;
+    },
+  },
+  // Currently duplicating to fix a where bug.
+  // Without this :installed_package:installed_packages: is not detect
+  // http://localhost:5173/guide/extensibility/packages.html#merging-and-order-of-precedence
+  {
+    regex: /:([\w+]*):([\w+]*):/g,
+    modifier: (match: RegExpMatchArray) => {
+      const m1 = match[1].replace(/_/g, " ");
+      const m2 = match[2].replace(/_/g, " ");
+      return `<Term term="${m1}" show="${m2}"/>`;
+    },
+  },
+  // Currently duplicating to fix a where bug.
+  // Without this :override_package: is not detect
+  // http://localhost:5173/guide/extensibility/packages.html#merging-and-order-of-precedence
+  {
+    regex: /:([\w+]*):/g,
+    modifier: (match: RegExpMatchArray) => {
+      const m1 = match[1].replace(/_/g, " ");
+      return `<Term term="${m1}"/>`;
+    },
+  },
```

Comment:
Not happy with this. Also no clue what is going on that requires this in order for all terms to be captured. Would love some additional eyes here